trailblazer/dto/__init__.py

```diff
@@ -0,0 +1,6 @@
+from trailblazer.dto.analyses_request import AnalysesRequest
```

Comment:
I do not know if these implicit imports in ini.py are a pattern we want? Would be nice to discus at a code review session.

Reply:
Yes, I think it is a good topic 👍 The upside is that we can do the imports like this
```python
from trailblazer.dto import (
    AnalysesRequest,
    AnalysisUpdateRequest,
    AnalysesResponse,
    AnalysisResponse,
)
```
Instead of having to specify the full paths like this
```python
from trailblazer.dto.analysis_response import AnalysisResponse
from trailblazer.dto.failed_jobs_request import FailedJobsRequest
from trailblazer.dto.failed_jobs_response import FailedJobsResponse
from trailblazer.dto.analysis_update_request import AnalysisUpdateRequest
from trailblazer.dto.analyses_response import AnalysesResponse
```

I like the first option because it is more concise, and modules using another module often import multiple things from it.

Reply:
Some advantages and disadvantages I see

Upsides
- More concise
- More readable
- Easier to refactor
- Better encapsulation (other modules do not need to know the internal file structure of other modules)

Downsides
- Namespace pollution
- Easier to run into circular imports (I guess the solution is to just use the full path for the import)
- Implicit dependencies

I think it can be beneficial for some types of modules, for instance pydantic models like this grouped in a single module. I think it is fine to use a mix and apply it wherever it benefits readability.

Reply:
I'll leave it as is for now.

Reply:
Rufff does not understand them:

```
trailblazer/dto/__init__.py:1:46: F401 [*] `trailblazer.dto.analyses_request.AnalysesRequest` imported but unused
trailblazer/dto/__init__.py:2:47: F401 [*] `trailblazer.dto.analyses_response.AnalysesResponse` imported but unused
trailblazer/dto/__init__.py:3:47: F401 [*] `trailblazer.dto.analysis_response.AnalysisResponse` imported but unused
trailblazer/dto/__init__.py:4:53: F401 [*] `trailblazer.dto.analysis_update_request.AnalysisUpdateRequest` imported but unused
trailblazer/dto/__init__.py:5:49: F401 [*] `trailblazer.dto.failed_jobs_request.FailedJobsRequest` imported but unused
trailblazer/dto/__init__.py:6:50: F401 [*] `trailblazer.dto.failed_jobs_response.FailedJobsResponse` imported but unused
trailblazer/services/__init__.py:1:51: F401 [*] `trailblazer.services.analysis_service.AnalysisService` imported but unused
trailblazer/services/__init__.py:2:46: F401 [*] `trailblazer.services.job_service.JobService` imported but unused
```